test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -43,7 +44,8 @@ def setup
       merchant_account_id: fixtures(:mercado_pago)[:merchant_account_id],
       fraud_scoring: true,
       fraud_manual_review: true,
-      payment_method_option_id: '123abc'
+      payment_method_option_id: '123abc',
+      notification_url: 'www.mercado-pago.com'
```

Comment:
Because this field isn't dependent on the processing mode (and because that's one of the remote tests that's currently failing), could you write a fresh remote test to just test this one new parameter?